build_tools/bazel/workspace.bzl

```diff
@@ -0,0 +1,123 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Helper functions for configuring IREE and dependent project WORKSPACE files."""
+
+load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
+load("@bazel_skylib//lib:paths.bzl", "paths")
+
+def configure_iree_submodule_deps(iree_repo_alias = "@", iree_path = "./"):
+    """Configure IREE all simple repository dependencies that come from submodules.
```

Comment:
I'm having trouble parsing this description. Reword?

Reply:
That's because it is not a real sentence :stuck_out_tongue: 

> Just scanned a few files...
> 
> Do you expect this to change the developer instructions for building IREE with Bazel/CMake?
> 
> e.g.
> [google.github.io/iree/get-started/getting-started-linux-bazel](https://google.github.io/iree/get-started/getting-started-linux-bazel)
> [google.github.io/iree/get-started/getting-started-linux-cmake](https://google.github.io/iree/get-started/getting-started-linux-cmake)
> 
> (the [python](https://google.github.io/iree/get-started/getting-started-python) docs are due for a total rewrite anyway :P)

Neither of those docs should require a change other than we might be able to get rid of some dependencies (which I'm planning to do there and in our docker containers as part of the follow-on cleanup to this).